include/mbedtls/check_config.h

```diff
@@ -626,7 +626,7 @@
 #if defined(MBEDTLS_SSL_PROTO_TLS1_3) && \
     ( ( !defined(MBEDTLS_HKDF_C) ) || \
       ( !defined(MBEDTLS_SHA256_C) && !defined(MBEDTLS_SHA384_C) ) || \
-      ( !defined(MBEDTLS_PSA_CRYPTO_C) ) )
+      ( !defined(MBEDTLS_USE_PSA_CRYPTO) ) )
```

Comment:
I think it's preferable to keep the dependency on `PSA_CRYPTO_C` - which means in `pk.h` making the new function (or part of it, as I'll suggest below) depend on `PSA_CRYPTO_C` rather than `USE_PSA_CRYPTO`.

Reply:
I don't think TLS 1.3 should depend on `USE_PSA_CRYPTO`, just `PSA_CRYPTO_C` should be enough - unless I missed something.